src/main/webapp/tests.html

```diff
@@ -18,5 +19,33 @@
   <script src="scripts/tests/perspective-test.js"></script>
   <script src="https://apis.google.com/js/api.js"></script>
   <script defer type="module" src="scripts/tests/comment-post-test.js"></script>
+  <script src="scripts/analyze.js"></script>
+  <script defer type="module" src="scripts/tests/analyze-test.js"></script>
```

Comment:
Not sure if this is the culprit but having all the tests in the same html might cause some issues. The scripts test could be potentially manipulating the dom at the same time. For your case, it doesn't seem like its touching the dom elements but maybe something else is happening.

Reply:
This worked! Thanks!